app/presenters/bidding_status_presenter/available.rb

```diff
@@ -19,6 +19,26 @@ def label
     'Open'
   end
 
+  def bid_label(user)
+    bid = highlighted_bid(user)
+    amount = Currency.new(bid.amount)
+
+    if bid.is_a?(NullBid)
+      I18n.t('bidding_status.available.bid_label.no_bids')
+    elsif auction.type == 'sealed_bid'
+      I18n.t('bidding_status.available.bid_label.sealed_bid',
+             amount: amount)
+    elsif auction.type == 'reverse' && bid.bidder == user
+      I18n.t('bidding_status.available.bid_label.reverse.vendor_winning',
+             amount: amount)
+    else
+      bidder_name = bid.bidder.name || bid.bidder.github_login
```

Comment:
When it's a reverse auction and the highlighted_bid is not the current user (diff between "Your winning bid" vs. "Winning bid")